mllib/src/test/scala/org/apache/spark/mllib/regression/LabeledPointSuite.scala

```diff
@@ -31,6 +31,11 @@ class LabeledPointSuite extends SparkFunSuite {
     }
   }
 
+  test("parse labeled points with whitespaces") {
+    val point = LabeledPoint.parse("(0.0, [1.0, 2.0])")
```

Comment:
Hm, doesn't this still yield ` 2.0` as a token during parsing? which wouldn't parse I'd imagine. Does this pass? in which case I'm missing something ...